packages/reakit/src/Sidebar/__tests__/Sidebar-test.tsx

```diff
@@ -0,0 +1,174 @@
+import * as React from "react";
```

Comment:
use toMatchSnapshot in here, just to make it more readable

Reply:
Done